src/style.css

```
@@ -0,0 +1,44 @@
+:root {
+  --starheight: 16px;
+  --starwidth: 16px;
```

Star has the same height and width. Only one variable needed.
```suggestion
  --star-dimensions: 16px;
```